src/cert_context.rs

```diff
@@ -119,6 +124,43 @@ impl CertContext {
         }
     }
 
+    /// Get certificate as PEM-formatted X509 certificate.
+    pub fn to_pem(&self) -> io::Result<String> {
+        unsafe {
+            let mut len = 0;
+            let ok = crypt32::CryptBinaryToStringA(
+                (*self.0).pbCertEncoded,
+                (*self.0).cbCertEncoded,
+                0 as winapi::DWORD,
```

Comment:
Could you explicitly pass `CRYPT_STRING_BASE64HEADER` here? It's equivalent but a bit more self documenting IMO.